pkg/ebpf/c/bpf_helpers.h

```diff
@@ -52,6 +52,11 @@ static int (*bpf_skb_set_tunnel_key)(void* ctx, void* key, int size, int flags)
 static unsigned long long (*bpf_get_prandom_u32)(void) = (void*)BPF_FUNC_get_prandom_u32;
 static u64 (*bpf_get_current_task)(void) = (void *) BPF_FUNC_get_current_task;
 
+static inline __attribute__((always_inline))
+u32 bpf_ntohl(u32 val) {
+  return __builtin_bswap32(val);
```

Comment:
Note: This is only correct for little endian systems. See https://github.com/torvalds/linux/blob/47ec5303d73ea344e84f46660fff693c57641386/tools/lib/bpf/bpf_endian.h#L50-L78

Reply:
Good catch 👍